### DISCUSSION TITLE
PCR test errors and basic person risk

### DISCUSSION THREAD

**Discussion by bluehat** [microCOVID/microCOVID#1041] [Q&A]
PCRs aren't great. [They have few false positives, but a high false negative rate.](https://www.acpjournals.org/doi/full/10.7326/M20-1495). 

> Over the 4 days of infection before the typical time of symptom onset (day 5), the probability of a false-negative result in an infected person decreased from 100% on day 1 to 68% on day 4. On the day of symptom onset, the median false-negative rate was 38%. This decreased to 20% on day 8 (3 days after symptom onset) then began to increase again, from 21% on day 9 to 66% on day 21. The false-negative rate was minimized 8 days after exposure—that is, 3 days after the onset of symptoms on average. As such, this may be the optimal time for testing if the goal is to minimize false-negative results. 

Do we need to adjust the basic person risk for the these false negatives? I don't see anything in the [COVID-19 Projections model which microCOVID is borrowing](https://covid19-projections.com/estimating-true-infections-revisited/#adjusted-test-positivity) that accounts for this.

**Comment by blanchardjeremy** [comment]
@beshaya @jy2wong @mattbell ?
